examples/c++/tmp006.cxx

```diff
@@ -58,8 +58,8 @@ int main(int argc, char **argv)
   // update and print available values every second
   while (run)
     {
-      // Print out temperature value in °C
-      std::cout << "Temperature: " << mySensor->getTemperature(true) << " °C"
+      // Print out temperature value in *C
+      std::cout << "Temperature: " << mySensor->getTemperature(true) << " *C"
```

Comment:
Agreed. Once this get merged into Master, Sonar will allow us to configure a [Quality Profile](https://docs.sonarqube.org/display/SONAR/Quality+Profiles) for the organization from its UI.